packages/quantic/force-app/main/default/lwc/exampleSearch/resultTemplates/caseResultTemplate.html

```diff
@@ -0,0 +1,32 @@
+<template>
+  <div class="lgc-bg slds-border_bottom">
+    <lightning-tile type="media">
+      <div class="slds-grid slds-wrap slds-m-vertical_x-small ">
+        <div class="slds-col slds-order_2 slds-large-order_1 slds-size_1-of-1 slds-large-size_3-of-12 slds-text-align_left" style="font-weight: 400; font-size:12px">
+          <span><lightning-icon slot="media" icon-name={icon} size="small"></lightning-icon></span>
+          <span class="slds-m-left_x-small">{result.raw.objecttype}</span>
+          <span class="slds-m-left_xxx-small" style="font-size: 10px; color: grey">[{result.raw.sfcasenumber}]</span>
+        </div>
+        <div class="slds-col slds-size_1-of-1 slds-large-size_6-of-12 slds-align-middle slds-large-order_2">
+          <template if:true={result.isRecommendation}><div><lightning-badge label="Recommended" class="recommended-badge" icon-name="utility:favorite"></lightning-badge></div></template>
```

Comment:
`label="Recommended"` should be localized. This comment applies to all result templates.